### PR TITLE
 mon/FSCommands: clean up FSCommands

### PR DESCRIPTION
**Pull request by rishabh-d-dave**
## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "pacific"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [ ] Doc update (no ticket needed)
  - [x] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [x] No impact that needs to be tracked
- Documentation (select at least one)
  - [ ] Updates relevant documentation
  - [x] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [x] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
</details>

### CODE REVIEW

**Review by leonid-s-usov — APPROVED** [review]
Looks good

File: src/mon/FSCommands.cc
Comment (leonid-s-usov) [inline src/mon/FSCommands.cc:1]: we could benefit from using `std::ostream`, `std::move`, `std::make_shared`

**Review by leonid-s-usov — APPROVED** [review]

File: src/mon/FSCommands.cc
Comment (mchangir) [reply src/mon/FSCommands.cc:1]: yeah ... explicit namespace scoping is actually a good practice even when the language itself provides a mechanism to do away with the explicit scoping

We are also using the boost library in some cases and the ceph namespace as well which contain similarly named classes.
It also becomes tedious to cross-verify which namespace does a class refer to every time an unscoped reference is found.

Comment (rishabh-d-dave) [reply src/mon/FSCommands.cc:1]: @leonid-s-usov 

Nice susggestion.

> we could benefit from using std::ostream, std::move, std::make_shared

I've imported `std::ostream` but I am letting `std::move` and `std::make_shared` stay unimported since they are not being import anywhere in `src/mon`, `src/osd` and `src/mds`.

**Review by leonid-s-usov — APPROVED** [review]

**Review by lxbsz — APPROVED** [review]

**Review by rishabh-d-dave — COMMENTED** [review]
CephFS integration testing was successful - https://tracker.ceph.com/projects/cephfs/wiki/Main#9-Oct-2023

**Review by ljflores — APPROVED** [review]
@rishabh-d-dave and I went through the rados suite run, and nothing is related:

https://pulpito.ceph.com/rishabh-2023-10-06_12:17:10-rados-rishabh-cephfs-mon-distro-default-smithi/

rados approved

**Comment by rishabh-d-dave** [comment]
Failure looks irrelevant - https://jenkins.ceph.com/job/ceph-api/62725. Running the test again.

**Comment by rishabh-d-dave** [comment]
jenkins test api

**Comment by rishabh-d-dave** [comment]
Waiting for Ceph API tests, I'll merge the PR after it passes.

**Comment by rishabh-d-dave** [comment]
Error is different this time but irrelevant

```
2023-10-09 18:27:29,835.835 INFO:__main__:ConnectionRefusedError: [Errno 111] Connection refused
```

```
2023-10-09 18:27:29,838.838 INFO:__main__:requests.exceptions.ConnectionError: HTTPSConnectionPool(host='172.21.5.33', port=7789): Max retries exceeded with url: /api/mgr/module (Caused by NewConnectionError(': Failed to establish a new connection: [Errno 111] Connection refused'))
```

https://jenkins.ceph.com/job/ceph-api/62905/

**Comment by rishabh-d-dave** [comment]
jenkins test api

**Pull request merged**
